pom.xml

```diff
@@ -37,10 +37,20 @@
               for the version of guava being upgraded to. -->
       <version>25.1-jre</version>
     </dependency>
+    <dependency>
+      <groupId>org.graalvm.nativeimage</groupId>
+      <artifactId>svm</artifactId>
+      <version>20.0.0</version>
```

Comment:
Same comment as I had for one of the previous PRs (it was unanswered)
Is it safe to hard-code the version? Quarkus is using env variable to take the graal version from the environment.
What if someone will a different or older graal version will try to run our code?

Reply:
@tomekl007 I [answered](https://github.com/datastax/java-driver/pull/1443#discussion_r432605072) this question on the prior PR, but just to re-state; this import is only used to provide the annotations necessary for the subs.  So long as the GraalVM impl respects those annotations there shouldn't be a version problem here.  I tested the prior version of this fix (using the same logic to express the dependency) back to Graal 19.2.1 and it worked without issue.